api-channel-jira/src/main/java/com/synopsys/integration/alert/api/channel/jira/distribution/search/JiraSearcher.java

```diff
@@ -149,13 +157,34 @@ protected JiraSearcher(String jiraProjectKey, JiraIssueAlertPropertiesManager is
     }
 
     private ProjectIssueSearchResult<String> createIssueResult(JiraSearcherResponseModel issue, ProjectIssueModel projectIssueModel) {
-        ExistingIssueDetails<String> issueDetails = createExistingIssueDetails(issue);
+        IssueCategory issueCategory = getIssueCategoryFromProjectIssueModel(projectIssueModel);
+        ExistingIssueDetails<String> issueDetails = createExistingIssueDetails(issue, issueCategory);
         return new ProjectIssueSearchResult<>(issueDetails, projectIssueModel);
     }
 
-    private ExistingIssueDetails<String> createExistingIssueDetails(JiraSearcherResponseModel issue) {
+    private ExistingIssueDetails<String> createExistingIssueDetails(JiraSearcherResponseModel issue, IssueCategory issueCategory) {
         String issueCallbackLink = JiraCallbackUtils.createUILink(issue);
-        return new ExistingIssueDetails<>(issue.getIssueId(), issue.getIssueKey(), issue.getSummaryField(), issueCallbackLink);
+        IssueStatus issueStatus = jiraIssueStatusCreator.createIssueStatus(issue, this::fetchIssueTransitions);
+        return new ExistingIssueDetails<>(issue.getIssueId(), issue.getIssueKey(), issue.getSummaryField(), issueCallbackLink, issueStatus, issueCategory);
+    }
+
+    private IssueCategory getIssueCategoryFromProjectIssueModel(ProjectIssueModel projectIssueModel) {
+        IssueCategory issueCategory = IssueCategory.BOM;
+        if (projectIssueModel.getVulnerabilityDetails().isPresent()) {
+            issueCategory = IssueCategory.VULNERABILITY;
+        } else if (projectIssueModel.getPolicyDetails().isPresent()) {
+            issueCategory = IssueCategory.POLICY;
+        }
+        return issueCategory;
     }
 
+    private IssueCategory getIssueCategoryFromComponentConcernType(ComponentConcernType componentConcernType) {
+        IssueCategory issueCategory = IssueCategory.BOM;
+        if (componentConcernType.equals(ComponentConcernType.VULNERABILITY)) {
+            issueCategory = IssueCategory.VULNERABILITY;
+        } else if (componentConcernType.equals(ComponentConcernType.POLICY)) {
+            issueCategory = IssueCategory.POLICY;
+        }
+        return issueCategory;
```

Comment:
Chose an arbitrary example, but I've noticed checks almost exactly like this littered through out the PR, perhaps we should have this as a converter or extractor in its own class?